tests/devices/test_default_qubit_jax.py

```diff
@@ -485,7 +485,7 @@ def circuit(weights):
             qml.templates.StronglyEntanglingLayers(weights, wires=[0, 1])
             return qml.expval(qml.PauliZ(0))
 
-        weights = jnp.array(qml.init.strong_ent_layers_normal(n_wires=2, n_layers=2))
+        weights = jnp.array(np.random.random(qml.templates.StronglyEntanglingLayers.shape(n_layers=2, n_wires=2)))

```

Comment:
A bit convoluted, but random number generation with jax is complicated, so I decided to parse here...